Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: destructible column pieces physics layer no longer interacts with PCs and NPCs

### Description (*)

PCs and NPCs were capable of colliding with column pieces (the Debris layer). I've adjusted the collision matrix to prevent this.

### Related Pull Requests

### Issue Number(s) (*)

MTT-2469 - https://jira.unity3d.com/browse/MTT-2469

Fixes issue(s):
### Manual testing scenarios

1. Start the game and go to the Boss's room
2. Make the Boss run into some columns - observe that the columns break, but the pieces just fall to the ground and afterwards are not interactable with the PCs and NPCs

### Questions or comments

### Contribution checklist
